### ISSUE TITLE
Follow up: update to newer Microsoft.NET.Test.Sdk once https://github.com/Microsoft/vstest/issues/373 is fixed

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#23254]
See https://github.com/dotnet/roslyn/pull/23253

**Comment by jcouv** [comment]
@tmat I'm still not getting source locations in unittest stacktraces (as shown below), even after the workaround you checked in. I'm using `Version 15.6.0 Preview 2.0 [27213.2.d15.6]`.

![image](https://user-images.githubusercontent.com/12466233/34064304-e8fcdb9e-e1ac-11e7-91fb-e1d2dade3883.png)

**Comment by jcouv** [comment]
Adding note to document current workaround to display line numbers in stacktraces:
1. Install 4.7.2
2. Modify "C:\Windows\Microsoft.NET\Framework\v4.0.30319\Config\machine.config" to add the `AppContextSwitchOverrides` flag:
``` 
  <runtime>
     <AppContextSwitchOverrides value="Switch.System.Diagnostics.IgnorePortablePDBsInStackTraces=false" />
  </runtime>
 
  <connectionStrings>
    ...
```

@tmat Where does the plan stand (short-term and long-term) to avoid this pain?

**Comment by tmat** [comment]
@noahfalk

**Comment by tmat** [comment]
At this point I think we should follow up with XUnit, NUnit and mstest to add this flag to their respective App.config files.

**Comment by jcouv** [comment]
I'm not sure where we're at with this issue. https://github.com/Microsoft/vstest/issues/373 was closed.
While working on Roslyn, I've not experienced the problem in a while, but that may be due to a workaround that Tomas put in place.

@tmat Could you confirm the current status and update the issue title accordingly? Or maybe this can be closed now? Thanks

**Comment by tmat** [comment]
There are still issues with the latest test SDK. We need to stay on the custom package for now. E.g. https://github.com/Microsoft/vstest/issues/1764, https://github.com/Microsoft/vstest/issues/792

**Comment by CyrusNajmabadi** [comment]
@tmat do we need to do anything here.  I don't even see any references to Microsoft.NET.Test in roslyn.
